infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/brokerphases/BrokerEnvironmentFactory.java

```diff
@@ -199,6 +190,23 @@ private Container newContainer(
     return container;
   }
 
+  protected List<String> getCommandLineArgs(RuntimeIdentity runtimeId) {
+    return new ArrayList<String>(
+        Arrays.asList(
+            "-push-endpoint",
+            cheWebsocketEndpoint,
+            "-runtime-id",
+            String.format(
+                "%s:%s:%s",
+                runtimeId.getWorkspaceId(),
+                MoreObjects.firstNonNull(runtimeId.getEnvName(), ""),
+                runtimeId.getOwnerId()),
+            "-cacert",
+            certProvisioner.isConfigured() ? certProvisioner.getCertPath() : "",
```

Comment:
How does golang's `flag` package handle the empty arg case? E.g. if certPath is not configured, wouldn't the arguments to the broker be
```
--push-endpoint <endpoint> --runtime-id <runtime-id> --cacert --registry-address
```
Is this a problem, or does golang handle it gracefully?

Reply:
I guess that it works fine as I just formatted that code and it had worked before my changes.

Reply:
Fair enough :)